### PR TITLE
 Add tooltip for long task name

### LINKED ISSUE #3: Add tooltip for long task name
When cursor hover on long tasks name inside the task selector, there should popup a tooltip to display the whole string.

This should be implemented with the [Tooltip](https://nextui.org/docs/components/tooltip) component of NextUI.

### PR DESCRIPTION
**Pull request by amelia2802**
- Added tooltip for long task name
- fixed #3 

## preview
image

### CODE REVIEW

File: .prettierrc.json
Comment (m4xshen) [inline .prettierrc.json:1]: This file should not be deleted

File: package.json
Comment (m4xshen) [inline package.json:1]: The next.js package version should also stay same.

**Review by m4xshen — CHANGES_REQUESTED** [review]

**Comment by amelia2802** [comment]
Hi, I've resolved those change requests.

File: yarn.lock
Comment (m4xshen) [inline yarn.lock:1]: I forgot to say that this also shouldn't change.

**Review by m4xshen — CHANGES_REQUESTED** [review]

**Comment by m4xshen** [comment]
@amelia2802 The feature is merged now. Thanks for the contribution!

**Pull request merged**